apps/agentic-chat/src/app/app.tsx

```diff
@@ -1,10 +1,34 @@
-import { wagmiConfig } from '../lib/wagmi-config';
+import { wagmiConfig, wagmiAdapter } from '../lib/wagmi-config';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 import { WagmiProvider } from 'wagmi';
 import { Dashboard } from './dashboard/page';
+import { createAppKit } from '@reown/appkit/react';
+import { networks } from '../lib/appkit';
+import { arbitrum } from '@reown/appkit/networks';
 
 const queryClient = new QueryClient();
 
+const metadata = {
+  name: 'Agentic Chat',
+  description: 'ShapeShift Agentic Chat',
+  url:
+    typeof window !== 'undefined'
+      ? window.location.origin
+      : 'https://chat.shapeshift.com',
```

Comment:
It probably won't be this, as I think we want to keep this product separate from ShapeShift, branding-wise. 

Fine for now though.

Reply:
Absolutely, no implications for the time being since domain does not exist!